phobos/cap-vars.yml

```diff
@@ -15,6 +15,8 @@ fiobench_pgnum: 2048
 fiobench_size: 100G
 ssd_bench: false
 
+bench_rgw_user_password: rgwbenchsecret
```

Comment:
Can we add this to perf-vars.yml as well?

Reply:
Definitely, i'll add it in